docs/applications/simple-bank-account/contract/register

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+${SCALAR_SDK_HOME}/client/bin/scalardl register-cert --properties ../conf/client.properties
+${SCALAR_SDK_HOME}/client/bin/scalardl register-contracts --properties ../conf/client.properties --contracts-file ../conf/contracts.toml
```

Comment:
```suggestion
${SCALAR_SDK_HOME}/bin/scalardl register-cert --properties ../conf/client.properties
${SCALAR_SDK_HOME}/bin/scalardl register-contracts --properties ../conf/client.properties --contracts-file ../conf/contracts.toml
```

As I mentioned in another comment, I think we need to fix the file path.

Reply:
Please check https://github.com/scalar-labs/scalardl/pull/61#discussion_r1687597593.